source/validate_id.cpp

```diff
@@ -997,13 +997,19 @@ bool idUsage::isValid<SpvOpVariable>(const spv_instruction_t* inst,
                           << "' is not a pointer type.";
     return false;
   }
-  if (opcodeEntry->numTypes < inst->words.size()) {
-    auto initialiserIndex = 4;
-    auto initialiser = module_.FindDef(inst->words[initialiserIndex]);
-    if (!initialiser || !spvOpcodeIsConstant(initialiser->opcode())) {
+  const auto initialiserIndex = 4;
```

Comment:
Since you are removing the only usage of `opcodeEntry` parameter that's passed to the function, you need to update the function definition header to:
```
bool idUsage::isValid<SpvOpVariable>(const spv_instruction_t* inst, const spv_opcode_desc) {
```